src/migrations/20221121114357-add-permission-for-environment-variants.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+          INSERT INTO permissions (permission, display_name, type)
+          values('UPDATE_FEATURE_ENVIRONMENT_VARIANTS', 'Update variants on environment', 'environment');
+
+          INSERT INTO role_permission (role_id, permission_id, environment)
```

Comment:
For every user can edit variants, give them the new permission in each environment - matches their old permission set